admin/src/pages/DashboardPage/components/LatestComments/index.tsx

```diff
@@ -0,0 +1,207 @@
+// @ts-nocheck
+
+import React, {memo, useRef, useMemo} from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators, compose} from 'redux';
+import isEqual from 'react-fast-compare';
+import {useQuery} from 'react-query';
+import {useHistory} from 'react-router-dom';
+import {isEmpty} from 'lodash';
+import {Box} from '@strapi/design-system/Box';
+import {
+  Layout,
+  HeaderLayout,
+  ActionLayout,
+  ContentLayout,
+} from '@strapi/design-system/Layout';
+import { styledLayout } from './styles';
+import {Table, Thead, Tbody, Tr, Th, Td} from '@strapi/design-system/Table';
+import {Typography} from '@strapi/design-system/Typography';
+import {VisuallyHidden} from '@strapi/design-system/VisuallyHidden';
+import {useNotifyAT} from '@strapi/design-system/LiveRegions';
+import {Grid} from '@strapi/design-system/Grid';
+
+import {
+  EmptyStateLayout,
+  SearchURLQuery,
+  LoadingIndicatorPage,
+  useTracking,
+  useNotification,
+  useRBAC,
+  useFocusWhenNavigate,
+  useQueryParams,
+} from '@strapi/helper-plugin';
+import getMessage from '../../../../utils/getMessage';
+import { fetchData } from '../../../Discover/utils/api';
+import pluginPermissions from '../../../../permissions';
+
+import getUrl from '../../../../utils/getUrl';
+import makeAppView from '../../../App/reducer/selectors';
+import DiscoverTableRow from '../../../Discover/components/DiscoverTableRow';
+import NoAcccessPage from '../../../NoAccessPage';
+import { PanelLayout, StyledHeader } from '../../styles';
+
+const COL_COUNT = 8;
+
+const LatestComments = ({ config }) => {
+  useFocusWhenNavigate();
+
+  const {push} = useHistory();
+  const {notifyStatus} = useNotifyAT();
+  const {trackUsage} = useTracking();
+  const trackUsageRef = useRef(trackUsage);
+  const toggleNotification = useNotification();
+  const [{query: queryParams}] = useQueryParams();
+  const _q = queryParams?._q || '';
+
+  const viewPermissions = useMemo(
+    () => ({
+      access: pluginPermissions.access,
+      moderate: pluginPermissions.moderate,
+      accessReports: pluginPermissions.reports,
+      reviewReports: pluginPermissions.reportsReview,
+    }),
+    [],
+  );
+
+  const {
+    isLoading: isLoadingForPermissions,
+    allowedActions: {
+      canAccess,
+      canModerate,
+      canAccessReports,
+      canReviewReports,
+    },
+  } = useRBAC(viewPermissions);
+
+  const {
+    isLoading: isLoadingForData,
+    data: {result, pagination = {}},
+    isFetching,
+  } = useQuery(
+    ['get-comments-data', queryParams, canAccess],
+    () => fetchData(queryParams, toggleNotification),
+    {
+      initialData: {},
+    },
+  );
+
+  const handleClickDisplay = (id) => {
+    push(getUrl(`discover/${id}`));
+  };
+
+  const isLoading = isLoadingForData || isFetching;
```

Comment:
```suggestion
  const isLoading = isLoadingForData || isFetching || isLoadingForPermissions;
```

To make compound flag, well, compound and in one place.